doc/source/cluster/kubernetes/user-guides/tpu.md

```diff
@@ -0,0 +1,71 @@
+(kuberay-tpu)=
+
+# Using TPUs with KubeRay
+
+This document provides tips on TPU usage with KubeRay.
+
+TPUs are available on Google Kubernetes Engine (GKE). To use TPUs with Kubernetes, configure
+both your Kubernetes setup and add additional values to your RayCluster configuration.
+Instructions for configuring TPUs on GKE can be found here:
+
+- {ref}`kuberay-gke-tpu-cluster-setup`
+
+## About TPUs
+
+TPUs are custom-designed AI accelerators which are optimized for training and inference of large AI models. A TPU host is a VM that runs on a physical computer connected to TPU hardware. TPU workloads can run on one or multiple hosts. A TPU Pod slice is a collection of chips all physically colocated and connected by high-speed inter chip interconnects (ICI). Single-host TPU Pod slices contain independent TPU VM hosts and communicate over the Data Center Network (DCN) rather than ICI interconnects. Multi-host TPU Pod slices contain two more interconnected TPU VM hosts. In GKE, multi-host TPU Pod slices run on their own node pools and are scaled atomically by node pools, rather than individual nodes. Both single-host and multi-host TPU Pod slices can be scaled seamlessly to multiple slices, enabling greater parallelism to support larger workloads.
+
+## Quickstart: Serve a Stable Diffusion model on GKE with TPUs
+
+After setting up a GKE cluster with TPUs and the Ray TPU initialization webhook, you're ready to begin running workloads on Ray with TPUs. {ref}`Serve a Stable Diffusion model on GKE with TPUs <kuberay-tpu-stable-diffusion-example>` shows how to serve a model with KubeRay on single-host TPUs.
+
+## Configuring Ray Pods for TPU usage
+
+Using any TPU accelerator requires specifying `google.com/tpu` resource `limits` and `requests` in the container fields of your `RayCluster`'s
+`workerGroupSpecs`. This resource specifies the number of TPU chips for GKE to allocate each Pod. KubeRay v1.1.0 adds a `numOfHosts`
+field to the RayCluster custom resource, specifying the number of TPU hosts to create per worker group replica. For multi-host worker groups,
+replicas are treated as Pod slices rather than individual workers, with `numOfHosts` worker nodes being created per replica.
+Additionally, GKE uses `gke-tpu` node selectors to schedule TPU Pods on the node matching the desired TPU accelerator and topology.
+
+Below is a config snippet for a RayCluster worker group with 2 Ray TPU worker Pods, each scheduled on its own GKE v4 TPU node belonging to the same TPU Pod slice.
```

Comment:
```suggestion
Below is a config snippet for a RayCluster worker group with 2 Ray TPU worker Pods. Ray schedules each worker group on its own GKE v4 TPU node belonging to the same TPU Pod slice.
```

Reply:
I'll add this change with the edit `Ray schedules each worker group` -> `Ray schedules each worker`